export/securedrop_export/disk/cli.py

```diff
@@ -430,7 +430,10 @@ def write_data_to_device(
             logger.debug("Copying file to {}".format(archive_target_dirname))
 
             subprocess.check_call(["cp", "-r", export_data, target_path])
-            logger.info("File copied successfully to {}".format(archive_target_dirname))
+            logger.info("File copied successfully to {}".format(target_path))
+
+            subprocess.check_call(["sudo", "chmod", "-R", "ugo+r", target_path])
```

Comment:
Do we not own the file at this point? If so, why do we need to sudo?

Reply:
This is the path `/media/user/$mountpoint/sd-export-yyyymmdd`, but you're right / if a user mounts with root it will fail at the `cp` stage so I probably don't need to hedge on that here

Reply:
Second question while I'm still thinking about this...if we're already recursive chmod-ing, do we need to even bother with the umask? i.e., can we just set permissions to 644 instead of just tweaking the read bit?

Reply:
Yep, this is the correct second question(s).  I can't remember why I switched from chmod to the umask approach when working on this, and having a bit of a facepalm since I think this means that the CI switch was not needed either, sorry. 

But the export structure is nested so I think we should still just flip the read bit

Reply:
Last update: I still want to leave the permissions test in the test suite, so the CI changes are helpful to the extent that they enable that, but you're right we can do all this without sudo, which was one of my goals in the switch from cryptsetup to udisks so I should have kept my eye on the prize :)  Thanks for the review.